package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@material-ui/core": "^4.10.0",
     "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
```

Comment:
Are we only getting the features we need from this alpha release? Are they available on non-alpha?

Reply:
I removed this library and styled the component myself. Looks about the same: 

![image](https://user-images.githubusercontent.com/12685223/88232175-a3fe3b00-cc32-11ea-8898-273446dd4c85.png)